updater.js

```diff
@@ -0,0 +1,252 @@
+var path = require('path')
+var fs = require('fs')
+var childProcess = require('child_process')
+var pkgUp = require('pkg-up')
+
+var BrowserslistError = require('./error')
+
+var UNKNOWN_VERSION = 'unknown version'
+var PACKAGE_CANIUSE = 'caniuse-lite'
+
+var packageManager = ''
+
+function updateDB () {
+  var lockfile = getLockfile()
+
+  fs.readFile(lockfile, 'utf8', function (readError, lockfileRaw) {
+    if (readError) {
+      throw readError
+    }
+
+    var currentVersion = getCurrentVersion(lockfileRaw)
+    var lastVersion = getLastVersion()
+
+    if (currentVersion === lastVersion) {
+      console.log(
+        'You have the latest version of ' + PACKAGE_CANIUSE + ' installed'
+      )
+      return
+    }
+
+    console.log(
+      'Current version: ' + currentVersion + '\n' +
+      'New version: ' + lastVersion + '\n' +
+      'Updating ' + PACKAGE_CANIUSE + '...'
+    )
+
+    fs.writeFile(lockfile, updateLockfile(lockfileRaw), function (writeError) {
+      if (writeError) {
+        throw writeError
+      }
+
+      var command = getPackageManager(lockfile) + ' install'
+      childProcess.exec(command, function (commandError) {
+        if (commandError) {
+          throw commandError
+        }
+
+        console.log(PACKAGE_CANIUSE + ' has been successfully updated')
+      })
+    })
+  })
+}
+
+function getLockfile () {
+  var packagePath = pkgUp.sync()
```

Comment:
You forget the case then `package.json` was not found

Reply:
I agree. This case needs to be covered.